src/evidence.cpp

```diff
@@ -616,6 +617,11 @@ void Courtroom::on_evidence_ok_clicked()
       set_evidence_page();
     }
   }
+
+  // Autosave private evidence
+  if (!current_evidence_global) {
+    evidence_save("inventories/autosave.ini");
```

Comment:
Why is this in a folder, with a plural if we can only have one, especially when the autosave file has such a non-descript name about its content. This just screams "I want to add more of this bs but sneak it into the file structure now so people won't stop me later."

Reply:
Because "Inventoriers" is the default folder it tries to access when you press the "Save" button @Salanto

Reply:
Cringe default path. Fine.